frontend/src/pages/article/components/author/author.tsx

```diff
@@ -0,0 +1,67 @@
+import { type FC } from 'react';
+
+import { Avatar, Icon } from '~/libs/components/components.js';
+
+import styles from './styles.module.scss';
+
+type Properties = {
+  name?: string;
+  followers?: number;
+  rating?: number;
+  publishedAt?: string;
+  readingTime?: string;
+  genre?: string;
+};
+
+const Author: FC<Properties> = ({
+  name = 'Charlie Culhane',
+  followers = 10,
+  rating = 700,
+  publishedAt = '2023-04-22',
+  readingTime = '7 min',
+  genre = 'Fiction',
+}) => {
```

Comment:
Do not forget to remove these stubs when you work with real data

Reply:
thanks, got it